cmd/notation/cert/add.go

```diff
@@ -28,6 +28,14 @@ func certAddCommand(opts *certAddOpts) *cobra.Command {
 			opts.path = args
 			return nil
 		},
+		Long: `Manage certificates in trust store
+
+Example - Add certificates with type "ca" to the trust store "acme-rockets":
```

Comment:
```suggestion
Example - Add certificates with type "ca" to the named store "acme-rockets":
```
"named store" is the term that we use in our spec. So maybe it's better to use it here?

Reply:
Agree